src/applications/vaos/tests/new-appointment/redux/actions.unit.spec.js

```diff
@@ -1,962 +0,0 @@
-    id: item.id,
-  })),
-);
-
-const userState = {
-  user: {
-    profile: {
-      facilities: [
-        {
-          facilityId: '983',
-          isCerner: false,
-        },
-        {
-          facilityId: '984',
-          isCerner: false,
-        },
-      ],
-    },
-  },
-};
-
-describe('VAOS newAppointment actions', () => {
-  it('should open form page', () => {
-    const action = openFormPage('test', 'uiSchema', 'schema');
-
-    expect(action).to.deep.equal({
-      type: FORM_PAGE_OPENED,
-      page: 'test',
-      uiSchema: 'uiSchema',
-      schema: 'schema',
-    });
-  });
-
-  it('should start new appointment flow', () => {
-    const action = startNewAppointmentFlow();
-
-    expect(action).to.deep.equal({
-      type: STARTED_NEW_APPOINTMENT_FLOW,
-    });
-  });
-
-  it('should hide podiatry appointment unavailable modal', () => {
-    const action = hidePodiatryAppointmentUnavailableModal();
-
-    expect(action).to.deep.equal({
-      type: FORM_HIDE_PODIATRY_APPOINTMENT_UNAVAILABLE_MODAL,
-    });
-  });
-
-  describe('routeToPageInFlow', () => {
-    it('should route to next page with string key', async () => {
-      const history = {
-        push: sinon.spy(),
-      };
-      const dispatch = sinon.spy();
-      const state = {};
-      const getState = () => state;
-      const data = {};
-
-      const thunk = routeToPageInFlow(testFlow, history, 'page2', 'next', data);
-      await thunk(dispatch, getState);
-
-      expect(dispatch.firstCall.args[0]).to.deep.equal({
-        type: FORM_PAGE_CHANGE_STARTED,
-        pageKey: 'page2',
-        data,
-      });
-      expect(dispatch.secondCall.args[0]).to.deep.equal({
-        type: FORM_PAGE_CHANGE_COMPLETED,
-        pageKey: 'page2',
-        pageKeyNext: 'page3',
-        direction: 'next',
-      });
-      expect(history.push.firstCall.args[0]).to.equal('/page3');
-    });
-
-    it('should route to next page with function', async () => {
-      const history = {
-        push: sinon.spy(),
-      };
-      const dispatch = sinon.spy();
-      const state = {};
-      const getState = () => state;
-
-      const thunk = routeToPageInFlow(testFlow, history, 'page2', 'next');
-      await thunk(dispatch, getState);
-
-      expect(history.push.firstCall.args[0]).to.equal('/page3');
-    });
-
-    it('should throw error for bad state', done => {
-      const history = {
-        push: sinon.spy(),
-      };
-      const dispatch = sinon.spy();
-      const state = {};
-      const getState = () => state;
-
-      const thunk = routeToPageInFlow(testFlow, history, 'page3', 'next');
-
-      thunk(dispatch, getState)
-        .then(() => {
-          done('Did not throw error for bad state');
-        })
-        .catch(e => {
-          expect(e.message).to.equal(
-            'Tried to route to page that does not exist',
-          );
-          done();
-        });
-    });
-
-    it('should route to previous page', async () => {
```

Comment:
We're not great about testing going back a page in the form flow, so it could be worth keeping these `routeToPageInFlow` tests.